SETUP.md

```diff
@@ -28,3 +37,5 @@ The repository files can be changed if the project requires it. If there is a pr
 - [ ] Update the `.gitignore` file
 - [ ] Update the pull request template
 - [ ] Update issue templates
+
+[^1]: Define how changes will be documented; e.g. `CHANGELOG.md` file, Releases section in GitHub, usage of additional tools, etc.
```

Comment:
Hmm, could we suggest (strongly advise) to use the releases section in GitHub? It's a common standard and the simplest way of doing it, I think we only added this to the general template since some teams used some additional tools for this.

Reply:
I agree, but I think we also have Changelog.md for most of our libs
But it is ok that we remove this

Reply:
I think we have both, either way it makes sense to suggest something that we consider to be the usual standard